php-checks/src/main/java/org/sonar/php/checks/formatting/ControlStructureSpacingCheck.java

```diff
@@ -94,15 +93,15 @@ private void checkSpaceBetweenKeywordAndNextNode(FormattingStandardCheck formatt
     AstNode keyword = controlStructure.getFirstChild(PHPKeyword.values());
     Token nextToken = keyword.getNextAstNode().getToken();
 
-    if (isType(nextToken, PHPPunctuator.LCURLYBRACE, PHPPunctuator.LPARENTHESIS) && isOnSameLine(keyword.getToken(), nextToken)) {
-      int nbSpace = getNbSpaceBetween(keyword.getToken(), nextToken);
-
-      if (nbSpace != 1) {
-        String endMsg = "between this \"" + keyword.getTokenOriginalValue() + "\" keyword and the opening "
-          + (isType(nextToken, PHPPunctuator.LPARENTHESIS) ? "parenthesis." : "curly brace.");
-        formattingCheck.reportIssue(buildIssueMsg(nbSpace, endMsg), keyword);
-      }
-    }
+//    if (isType(nextToken, PHPPunctuator.LCURLYBRACE, PHPPunctuator.LPARENTHESIS) && isOnSameLine(keyword.getToken(), nextToken)) {
+//      int nbSpace = getNbSpaceBetween(keyword.getToken(), nextToken);
+//
+//      if (nbSpace != 1) {
+//        String endMsg = "between this \"" + keyword.getTokenOriginalValue() + "\" keyword and the opening "
+//          + (isType(nextToken, PHPPunctuator.LPARENTHESIS) ? "parenthesis." : "curly brace.");
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) This block of commented-out lines of code should be removed. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3ACommentedOutCodeLine)